tests/test_nginx_config_reloader.py

```diff
@@ -26,7 +35,13 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.source, ignore_errors=True)
         shutil.rmtree(self.dest, ignore_errors=True)
-        shutil.rmtree(nginx_config_reloader.BACKUP_CONFIG_DIR, ignore_errors=True)
+        shutil.rmtree(self.backup, ignore_errors=True)
+        try:
+            os.unlink(self.mag_conf)
+            os.unlink(self.mag1_conf)
+            os.unlink(self.mag2_conf)
+        except OSError:
```

Comment:
These should go into their own `try/except` statements. If the first `os.unlink` fails it will never try the second or third anymore.